telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
+message NameIDMessage { //Send the device ID with its corresponding name
+	optional uint32 id = 1;
+	optional string name = 2 [(nanopb).max_size = 32];
+}
+
+message StatisticalAggregate {
+  optional uint32 numSamples = 1;
+  optional uint32 min = 2;
+  optional uint32 max = 3;
+  optional uint32 mean = 4;
+  optional uint32 stdev = 5;
+}
+
+message GeneralReading { //this might want a more specific name?
+	enum READING_TYPE {
+		VOLTAGE = 1;
+		CURRENT = 2;
+		TEMPERATURE = 3;
+	}
+	optional READING_TYPE type = 1;
```

Comment:
Actually, if the NameIDMessage is now going to include metadata like the reading type, would it also make sense to send information like the expected time between samples, sensor limits, etc.? I assumed that the NameIDMessage would have been used to update the config entries, but that type of message seems like it would become rather memory-intensive, given the amount of information each config entry would need. Would it just be easier to update the config file itself and remove the NameIDMessages entirely, or is it better to keep some means of communication from the telemetry board to the viewer?